ElfUtils/LinuxMap.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ElfUtils/LinuxMap.h"
+
+#include <absl/strings/str_format.h>
+#include <absl/strings/str_split.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include <filesystem>
+
+#include "ElfUtils/ElfFile.h"
+#include "OrbitBase/File.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/SafeStrerror.h"
+
+namespace elf_utils {
```

Comment:
Based on the discussions, the namespace should be `orbit_elf_utils` (unlike the dir name).

Reply:
done.